ui/components/DataTable.tsx

```diff
@@ -134,24 +138,44 @@ function UnstyledDataTable({
             ) : (
               <EmptyRow colSpan={fields.length}>
                 <TableCell colSpan={fields.length}>
-                  <span style={{ fontStyle: "italic" }}>No rows</span>
+                  <Flex center align>
+                    <Icon
+                      color="neutral20"
+                      type={IconType.RemoveCircleIcon}
+                      size="base"
+                    />
+                    <Spacer padding="xxs" />
+                    <Text color="neutral30">No data</Text>
+                  </Flex>
                 </TableCell>
               </EmptyRow>
             )}
           </TableBody>
         </Table>
       </TableContainer>
+      <Spacer padding="xs" />
+      {/* optional pagination component */}
+      {children}
     </div>
   );
 }
 
 export const DataTable = styled(UnstyledDataTable)`
   h2 {
+    font-size: 14px;
+    font-weight: 600;
```

Comment:
I have played around with this in the inspector and for me this only renders bold (> 600) or regular (<600) – is semi-bold Proxima Nova not getting imported somewhere?

Reply:
Hmmmm...I feel like I've seen that before, but I just tested it here's the screenshots:

font-weight 800:
![image](https://user-images.githubusercontent.com/65822698/152192719-39d93313-0485-42ca-804f-c5f9b0ba9ab8.png)

font-weight 600:
![image](https://user-images.githubusercontent.com/65822698/152192803-717ada00-5ca3-4f31-ad27-8cb637cc5bae.png)

Reply:
It DEFINITELY changes but it is subtle for sure